libs/elements/cards/src/lib/components/elewa-info-card/elewa-info-card.component.scss

```diff
@@ -3,35 +3,34 @@
     flex-direction: row;
 }
 
-.info-card {
-    height: 180px;
-    width: 630px;
-}
-
-.icon-img {
+.icon {
     width: 52px;
     height: 54px;
     margin-right: 57px;
 }
 
 .info-card-title {
-    font-size: 42px;
+    font-size: 18px;
     font-family: "DMSans-Medium", Arial, Helvetica, sans-serif;
```

Comment:
Add to variables.scss and reference it from there.

Reply:
@Beulah-Matt I have made the changes